src/WebApp/AIOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace eShop.WebApp;
+
+public class AIOptions
+{
+    /// <summary>Settings related to the use of OpenAI.</summary>
+    public OpenAIOptions OpenAI { get; set; } = new();
+}
+
+public class OpenAIOptions
+{
+    /// <summary>OpenAI API key for accessing embedding LLM.</summary>
+    public string? ApiKey { get; set; }
+
+    /// <summary>Optional endpoint for which OpenAI API to access.</summary>
+    public string? Endpoint { get; set; }
+
+    /// <summary>The name of the chat model to use.</summary>
+    public string ChatName { get; set; } = "gpt-3.5-turbo-16k";
```

Comment:
Same comment as above about the endpoint. Also this is the model name in the case of OpenAI, but the deployment name in the case of Azure OpenAI.

Reply:
I'll add a remark here that needs to be "deployment name" if you are using Azure OpenAI.